### PR TITLE
Add github action

### PR DESCRIPTION
**Pull request by Poggicek**
Add automatic building and deploying when you push a commit with a tag "v*". It uses some shitty free github virtual servers so you don't have to pay shit, and it's fast Pog.

### CODE REVIEW

**Review by jooaooalmeida — APPROVED** [review]
Worked during the tests on the fork.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
```
Comment (tabarra) [inline .github/workflows/build.yml:13]: needs to be built with node 12, since fxserver uses node 12

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+      - run: npm i
+      - run: npm run build
+      - uses: papeloto/action-zip@v1
+        with:
+          files: dist/
+          recursive: true
+          dest: monitor.zip
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
```
Comment (tabarra) [inline .github/workflows/build.yml:30]: this repo seems kinda small for something that is super common... perhaps there is a reason for that?  
May be a good idea to try to find another one.

**Comment by Poggicek** [comment]
going to experiment with different release script

**Pull request closed without merging**